src/style.css

```
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  display: block;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.big {
+  height: 70px;
+}
+
+.small {
+  height: 42px;
+}
```

you can make it like that and we will have more reusable stuff :)
And one more point - we use modifiers with `--`. Like `form--big` `form--small`  and so on. You will learn about it later in the topic BEM
```suggestion
.form {
  display: block;
  height: 70px;
  justify-content: center;
  margin: 20px auto 0;
}

.form--small {
  height: 42px;
}
```